### PR TITLE
[1.20] Update all Chinese translation (zh_cn) for Animals Dictionary

### PR DESCRIPTION
**Pull request by NumberSir**

### CODE REVIEW

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    秃鹰是在树木丰富的山地群系中出
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt:4]: 秃鹰→白头海雕

**Review by mc-kaishixiaxue — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt (hunk 1 above)
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
    白头海雕是在树木丰富的山地群系中出
```

请和 [zh_cn.json](https://github.com/AlexModGuy/AlexsMobs/blob/1.20/src/main/resources/assets/alexsmobs/lang/zh_cn.json) 相对应，其他文件需要检查。
Please correspond to zh_cn.json, other files need to be checked.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<NEWLINE>
+兜帽可以戴在秃鹰头上，戴着兜帽的秃鹰在
+<NEWLINE>
+从手套放出时将遵循主人的直接命令。
+<NEWLINE>
+    这意味着秃鹰可以有效地由其主人指挥。
+<NEWLINE>
+如果秃鹰飞离其主人超过150个方块，它将试
+<NEWLINE>
+图返回。要停止控制秃鹰，只需下蹲即可。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
图返回。要停止控制秃鹰，只需潜行即可。
```

`sneak` 要翻译为 `潜行`。
`sneak` is to be translated as `潜行`.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    抹香鲸是一种庞大的、中性的哺乳
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]:
```suggestion
    抹香鲸是一种庞大的、中立的哺乳
```
`neutral`通常翻译为`中立`，就像末影人一样。其他文件请自行排查。
`neutral` usually translates to `中立`, as Enderman. Please check the other files by yourself.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+<NEWLINE>
+    在强烈雷暴期间，抹香鲸可能会被
+<NEWLINE>
+冲上海滩。它们无法凭自己的力量返回
+<NEWLINE>
+海中，如果被玩家平安无事地推回水中，
+<NEWLINE>
+它们可能会决定奖励其的救援者。
+<NEWLINE>
+    奖励一般为麝香，是抹香鲸胃部分
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]:
```suggestion
    奖励一般为龙涎香，是抹香鲸胃部的分
```
`麝香`明显与抹香鲸无关，后面一句少了`的`变得不通顺。
`麝香` clearly has nothing to do with Cachalot Whale. The latter sentence becomes incoherent without `的`.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+<NEWLINE>
+一旦发现一个洞穴，定位器将朝着洞穴
+<NEWLINE>
+的方向发射声波。注意，回声定位器的
+<NEWLINE>
+使用次数是有限的。
+<NEWLINE>
+    如果用4个末影珍珠和回声定位器
+<NEWLINE>
+一起合成，可以得到1个末地定位器，
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]:
```suggestion
一起合成，可以得到1个末影定位器，
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+<NEWLINE>
+它可以在靠近要塞时显示末影传送门的
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]:
```suggestion
它可以在靠近要塞时显示末地传送门的
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 7]
```diff
@@ -0,0 +1,60 @@
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    秃鹰是在树木丰富的山地群系中出
+<NEWLINE>
+没的大型、中性的猛禽。它们经常前往附
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
没的大型、中立的猛禽。它们经常前往附
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/frilled_shark.txt [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+<NEWLINE>
+    如果玩家获得了一些鲨鱼牙齿，可
+<NEWLINE>
+以将它们与其他一些水生物品一起合成
+<NEWLINE>
+出深澜之盾，比普通的盾牌更耐用，在
+<NEWLINE>
+阻挡攻击时也会对攻击者施加“血液流失”
+<NEWLINE>
+效果。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/frilled_shark.txt]:
```suggestion
阻挡攻击时也会对攻击者施加“失血”效果。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/leafcutter_ant.txt [hunk 9]
```diff
@@ -0,0 +1,62 @@
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    叶切蚁是一种小巧的昆虫，常常可
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/leafcutter_ant.txt]:
```suggestion
    切叶蚁是一种小巧的昆虫，常常可
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mantis_shrimp.txt [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+<NEWLINE>
+    这对拳头的速度足以将敌人打飞出
+<NEWLINE>
+去，并撞破方块。
+<NEWLINE>
+    螳螂虾是中立的，但激怒它们是不
+<NEWLINE>
+明智的。螳螂虾是聪明的掠食者，它的
+<NEWLINE>
+食谱包括热带鱼、乌贼、守卫者和它们
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mantis_shrimp.txt]:
```suggestion
食谱包括热带鱼、鱿鱼、守卫者和它们
```
原版鱿鱼？
Vanilla  squid?

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mantis_shrimp.txt [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+<NEWLINE>
+过 5 分钟就会开始变干。潜行并对驯
+<NEWLINE>
+服的螳螂虾使用水桶可以防止它变干，
+<NEWLINE>
+这将使得它们可以无限期地离开水生活
+<NEWLINE>
+    可以命令驯服的螳螂虾跟随、坐下
+<NEWLINE>
+、闲逛或破坏方块。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mantis_shrimp.txt]:
```suggestion
    可以命令驯服的螳螂虾跟随、坐下、
<NEWLINE>
游荡或破坏方块。
```
顿号不能写在行的开头。
The enumeration comma cannot be written at the beginning of a line.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+<NEWLINE>
+蘑菇类型相关的生物群系。如果是红色
+<NEWLINE>
+或棕色的蘑菇，会变为蘑菇岛；如果是
+<NEWLINE>
+绯红菌或诡异菌，则会变为绯红森林或
+<NEWLINE>
+诡异森林。
+<NEWLINE>
+    可以喂食蘑菇君一个毒土豆来撤销
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt]:
```suggestion
    可以喂食蘑菇君一个毒马铃薯来撤销
```

原版物品
Vanilla item

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/sea_bear.txt [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    海熊是真实存在的。如果你想远离
+<NEWLINE>
+海熊，那么你不该做这些事情：
+<NEWLINE>
+    首先，不要吹竖笛。
+<NEWLINE>
+    别戴宽边帽，像傻瓜那样。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/sea_bear.txt]:
```suggestion
    别像傻瓜那样戴阔边帽。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/sea_bear.txt [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+<NEWLINE>
+    或者小丑鞋，或者裙子。
+<NEWLINE>
+    而且永远别像黑猩猩那样挠痒痒！
+<NEWLINE>
+    如果你知道一定会有海熊来吃你，
+<NEWLINE>
+那就用棍子在沙子上画一个反海熊圈。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/sea_bear.txt]:
```suggestion
那就用木棍在沙子上画一个反海熊圈。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/tiger.txt [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+<NEWLINE>
+    老虎是一种大型猫科动物，可以在
+<NEWLINE>
+竹林内及其周围找到。它们天生警觉，
+<NEWLINE>
+会对靠近的外来者发动攻击。
+<NEWLINE>
+    老虎是优秀的猎手，它们非常强壮
+<NEWLINE>
+，能够从许多方块以外跃过来碾碎它的
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/tiger.txt]:
```suggestion
    老虎是优秀的猎手，它们非常强壮，
<NEWLINE>
能够从许多方块以外跃过来碾碎它的
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+<NEWLINE>
+以遇到它们。这种奇怪的哺乳动物的脚
+<NEWLINE>
+上有毒刺，用来对任何攻击者施加中毒
+<NEWLINE>
+效果。
+<NEWLINE>
+    鸭嘴兽的嗅觉、听觉和视力都很弱
+<NEWLINE>
+，这使得它不得不依靠电力来捕猎和导
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt]:
```suggestion
    鸭嘴兽的嗅觉、听觉和视力都很弱，
<NEWLINE>
这使得它不得不依靠电力来捕猎和导
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+<NEWLINE>
+    在寻找食物时，鸭嘴兽会挖出一些
+<NEWLINE>
+黏土球和蛆，玩家可以拾取这些东西。
+<NEWLINE>
+如果喂它整个红石块，挖出的物品数量
+<NEWLINE>
+会大大增加。与其他小型水生生物一样
+<NEWLINE>
+，可以用水桶运输鸭嘴兽。可以用龙虾
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt]:
```suggestion
会大大增加。与其他小型水生生物一样，
<NEWLINE>
可以用水桶运输鸭嘴兽。可以用龙虾
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+<NEWLINE>
+尾来繁殖鸭嘴兽，这是它最喜欢的食物
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/platypus.txt]:
```suggestion
尾来繁殖鸭嘴兽，这是它最喜欢的食物。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 19]
```diff
@@ -0,0 +1,60 @@
...
+<NEWLINE>
+空盘旋，挥舞着爪子攻击，并循环往复。
+<NEWLINE>
+    由于秃鹰喜欢吃鱼，可以用鱼油引诱
+<NEWLINE>
+和驯服秃鹰。与许多其他可驯服的生物一
+<NEWLINE>
+样，被驯服的秃鹰可以停留、跟随或徘徊
+<NEWLINE>
+，但它们也可以用于打猎。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
样，被驯服的秃鹰可以停留、跟随或游荡，
<NEWLINE>
但它们也可以用于打猎。
```
逗号不能写在行的开头。
The comma cannot be written at the beginning of a line.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+<NEWLINE>
+被动的。
+<NEWLINE>
+    从覆盖它们的菌丝皮毛中，偶尔会
+<NEWLINE>
+长出一颗蘑菇。喂食它更多相同的蘑菇
+<NEWLINE>
+，就会有更多的蘑菇在蘑菇君身上生根
+<NEWLINE>
+发芽。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt]:
```suggestion
长出一棵蘑菇。喂食它更多相同的蘑菇
<NEWLINE>
就会有更多蘑菇在蘑菇君身上生根发芽。
```
量词不对。
This quantifier is wrong.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 21]
```diff
@@ -0,0 +1,60 @@
...
+<NEWLINE>
+    要进行鹰猎，需要先制作猎鹰手套。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
    要进行鹰猎，需要先合成猎鹰手套。
```
`craft` 要翻译为 `合成`。
`craft` is to be translated as `合成`.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt [hunk 22]
```diff
@@ -0,0 +1,60 @@
...
+<NEWLINE>
+出秃鹰，秃鹰将飞向该敌对生物进行攻击，
+<NEWLINE>
+如果秃鹰存活下来，就会返回手中。
+<NEWLINE>
+    如果没有瞄准敌对生物，秃鹰只会简
+<NEWLINE>
+单地跳离手套。
+<NEWLINE>
+    为了提高鹰猎技能，可以制作猎鹰兜帽。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]:
```suggestion
    为了提高鹰猎技能，可以合成猎鹰兜帽。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt [hunk 23]
```diff
@@ -0,0 +1,69 @@
...
+<NEWLINE>
+海中，如果被玩家平安无事地推回水中，
+<NEWLINE>
+它们可能会决定奖励其的救援者。
+<NEWLINE>
+    奖励一般为麝香，是抹香鲸胃部分
+<NEWLINE>
+泌物。麝香可以与渔夫交换大量的绿宝
+<NEWLINE>
+石，也可以用来制作回声定位器。还可
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]:
```suggestion
石，也可以用来合成回声定位器。还可
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt [hunk 24]
```diff
@@ -0,0 +1,52 @@
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+<NEWLINE>
+    蘑菇君是一种奇怪真菌生物，被目
+<NEWLINE>
+击于蘑菇岛。这些奇怪的生物是守卫者
+<NEWLINE>
+和炽足兽的亲戚，外表与他们相似。与
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt]: 该文件都用`它们`，为什么这里用`他们`？
This file uses `它们`, so why do you use the word `他们` here?

**Review by mc-kaishixiaxue — CHANGES_REQUESTED** [review]

**Review by mc-kaishixiaxue — CHANGES_REQUESTED** [review]
审查已完成。另外，以下几个在`zh_cn`文件夹中的文件还没有翻译，其对应的英文可能不一样：
The review has been completed. In addition, the following files in `zh_cn` folder have not been translated, and their English texts may be different:
1. cockroach.txt
2. crow.txt
3. elephant.txt
4. seal.txt

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt (hunk 2 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]: fixed in:
- bald_eagle

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt (hunk 3 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]: fixed in:
- bald_eagle
- cachalot_whale
- frilled_shark

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt (hunk 4 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]: fixed:
`麝香` -> `龙涎香`

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt (hunk 5 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]: fixed

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt (hunk 6 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cachalot_whale.txt]: fixed

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt (hunk 12 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt]: fixed

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt (hunk 24 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt]: fixed

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt (hunk 1 above)
Comment (NumberSir) [reply src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/bald_eagle.txt]: fixed in:
- bald_eagle
- cachalot_whale
- leafcutter_ant

**Comment by NumberSir** [comment]
> 审查已完成。另外，以下几个在`zh_cn`文件夹中的文件还没有翻译，其对应的英文可能不一样： The review has been completed. In addition, the following files in `zh_cn` folder have not been translated, and their English texts may be different:
> 
> 1. cockroach.txt
> 2. crow.txt
> 3. elephant.txt
> 4. seal.txt

感谢审查，提到的问题已修复。质量过低的译文已改用我在 1.19.3 版本分支提交过的翻译版本：#1600

---

Thank you for the review. The issues mentioned have been fixed. The translation with low quality has been changed to the version I commited in version 1.19.3 branch: #1600

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cockroach.txt [hunk 25]
```diff
@@ -1,11 +1,48 @@
...
-Although naturally skittish around people, cockroaches can be fed bread or sugar to make them more comfortable. Cockroaches are considered a nuisance as they eat any dropped food item, making it dangerous to leave food unguarded in caves.
-Covering a Cockroach in Mimicream from a Mimicube can result in a roach with a very interesting color scheme. It is also said they have a fondness for Maracas.
-<NEWLINE>
-Cockroaches are the natural prey of Cave Centipedes, which will seek them out from far away. When injured gravely, it is possible for a roach to loose its head, yet it seems they are fully capable of functioning in this beheaded form.
-On death, cockroaches have a chance to drop Cockroach Wing Fragments. Nine of these can be used to create a full Cockroach Wing, which in turn can be used to brew a rare potion of Bug Pheromones.
-These pheromones will allow one to go undetected near any hostile arthropod, but not completely invisible. Cockroaches can be bred with sugar.
\ No newline at end of file
+    蟑螂是生成于地下的一种脏脏虫，
+<NEWLINE>
+他们会躲避阳光。这种虫子避光，以免
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/cockroach.txt:6]: 代词错误，该文件出现了3个`他们`。
Wrong pronoun, this file appears 3 `他们`.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/crow.txt [hunk 26]
```diff
@@ -1,11 +1,50 @@
...
+<NEWLINE>
+，它会从贴着展示框的一面把物品放进
+<NEWLINE>
+容器。
+<NEWLINE>
+    当乌鸦设定为跟随状态时会绕着主
+<NEWLINE>
+人飞行，并且会尝试坐在主人的肩膀上
+<NEWLINE>
+    下蹲时可以让肩膀上的乌鸦下来。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/crow.txt:44]:
```suggestion
    潜行可以让肩膀上的乌鸦下来。
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/seal.txt [hunk 27]
```diff
@@ -1,7 +1,38 @@
...
+<NEWLINE>
+    如果一只海豹受到攻击，附近所有
+<NEWLINE>
+海豹都会试图逃到水中，在那里它们更
+<NEWLINE>
+敏捷。
+<NEWLINE>
+    当海豹在陆地上晒太阳时，用3条
+<NEWLINE>
+鱼去喂他可以引发你和海豹之间的原始
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/seal.txt:26]:
```suggestion
    当海豹在陆地上晒太阳时，喂给它
<NEWLINE>
3 条鱼可以发起你和海豹之间的原始
```

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/frilled_shark.txt [hunk 28]
```diff
@@ -0,0 +1,35 @@
...
+<NEWLINE>
+    皱鳃鲨是在深海中发现的懒洋洋的
+<NEWLINE>
+活化石。这种中立鲨鱼大部分时间都在
+<NEWLINE>
+捕猎它们最喜欢的猎物：水滴鱼和鱿鱼。
+<NEWLINE>
+    在低光照度的深海，皱鳃鲨用它能
+<NEWLINE>
+反光的牙齿吸引本能向光的乌贼来捕猎
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/frilled_shark.txt:13]: `乌贼`是原版的`鱿鱼`还是本模组的`大王乌贼`？
Is `乌贼` vanilla `Squid` or this mod's `Giant Squid`?

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt [hunk 29]
```diff
@@ -0,0 +1,50 @@
...
+<NEWLINE>
+的光束。它们会借此长出更多该类型的
+<NEWLINE>
+蘑菇，如果蘑菇盖过了菌丝体，就会变
+<NEWLINE>
+成一个巨型蘑菇。这将耗尽蘑菇君的精
+<NEWLINE>
+力并减少生长于其上的蘑菇。如果被杀
+<NEWLINE>
+时背上有 5 个蘑菇，蘑菇君将发出剧
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/mungus.txt:34]: `蘑菇`请统一量词。
`蘑菇` Please standardize the quantifier.

File: src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/seal.txt [hunk 30]
```diff
@@ -1,7 +1,38 @@
...
+<NEWLINE>
+交易：为了换取这 3 条鱼，海豹会在
+<NEWLINE>
+海床上找寻掉落物作为回报。尽管这种
+<NEWLINE>
+回报往往是毫无价值的垃圾，比如海带
+<NEWLINE>
+或沙子，但偶尔可能会是比较罕见的物
+<NEWLINE>
+品比如鲨鱼利齿、锥形鳞甲或鹦鹉螺壳。
```
Comment (mc-kaishixiaxue) [inline src/main/resources/assets/alexsmobs/book/animal_dictionary/zh_cn/seal.txt:36]:
```suggestion
或沙子，但有时也可能是比较罕见的物
<NEWLINE>
品，比如鲨鱼利齿、锥形鳞甲或鹦鹉螺壳。
```

**Review by mc-kaishixiaxue — CHANGES_REQUESTED** [review]

**Pull request merged**